saleor/dashboard/product/views.py

```diff
@@ -30,9 +30,10 @@ def product_class_list(request):
         (pc.pk, pc.name, pc.has_variants, pc.product_attributes.all(),
          pc.variant_attributes.all())
         for pc in classes.object_list]
+
     ctx = {'form': form, 'product_classes': classes}
-    return TemplateResponse(
-        request, 'dashboard/product/product_class/list.html', ctx)
+    class_list_tmpl = 'dashboard/product/product_class/list.html'
```

Comment:
Instead of introducing different variable name for template path in every view, I would rather expect simply a `template` variable everywhere. It's more consistent.